ext/rp/pico.cpp

```diff
@@ -0,0 +1,20 @@
+#include "pico.h"
+#include "hardware/resets.h"
+
+#include <modm/platform/core/resets.hpp>
+
+/**
+ * Pico SDK compatibility for tinyusb
+ */
+
+extern "C" void reset_block(uint32_t blocks) {
+	modm::platform::Resets::reset(blocks);
+}
+extern "C" void unreset_block_wait(uint32_t blocks) {
+	modm::platform::Resets::unresetWait(blocks);
+}
+
+extern "C" void panic(const char *fmt, ...) {
+	// @todo any modm implementation? assert?
+	//va_list args;
```

Comment:
Maybe something like this taken from [our TinyUSB port](https://github.com/modm-io/modm/blob/develop/ext/hathach/tusb_port.cpp.in#L66-L80) (`with_debug = env.has_module(":debug")`):
```suggestion
%% if with_debug
	va_list va;
	va_start(va, fmt);
	modm::log::debug.vprintf(fmt, va);
	va_end(va);
%% endif
modm_assert(0, "pico", "Pico-SDK panic!");
```
The second modm_assert string does not support formatting…